README.md

```diff
@@ -1,6 +1,9 @@
 # cidr-filtering-cni
 
-This plugin allows user to define allow list for ingress and egress packet filtering for a Kubernetes pod.
+This CNI plugin isolates pods from external endpoints. By default, all ingress and egress traffic is denied.
```

Comment:
I would try to avoid using terms such as endpoints (which have a meaning of its own on kubernetes), and also external (external to what ?... the cluster ?.. the pod ? ... ).

How about something like: "This CNI plugin allows the user to control traffic flow at the IP address or port level (OSI layer 3 or 4) for particular applications in the Kubernetes cluster, thus specifying how a [pod](https://kubernetes.io/docs/concepts/workloads/pods/) is allowed to communicate with various network "entities".